.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -1,7 +0,0 @@
-<component name="InspectionProjectProfileManager">
```

Comment:
Are these changes intentional?

Reply:
Another good catch! Reverted in d8061682ba9f6d537e834712f013401b88a86f45